examples/user_interfaces/toolmanager.py

```diff
@@ -83,10 +114,21 @@ def set_lines_visibility(self, state):
 fig.canvas.manager.toolbar.add_tool('zoom', 'foo')
 
 # Remove the forward button
-fig.canvas.manager.toolmanager.remove_tool('forward')
+tool_mgr.remove_tool('forward')
 
 # To add a custom tool to the toolbar at specific location inside
 # the navigation group
 fig.canvas.manager.toolbar.add_tool('Hide', 'navigation', 1)
 
+for i, c in enumerate(['yellowgreen', 'forestgreen']):
+    sidebar = fig.canvas.manager._get_toolbar()
```

Comment:
@OceanWolf sorry for the delay, I have been really busy at work.

I don't think we need nor want more than one `toolmanager` per `figuremanager`

The possibility of more than one `toolbar` seems like a good idea, but I don't think many people will use it.
So instead of polluting with a new method I would suggest to stick to your proposal of
`fig.canvas.manager.get_backend().Toolbar(fig.canvas.manager.toolmanager)`